docs/api/mlrun.utils.rst

```diff
@@ -1,6 +0,0 @@
-mlrun.utils
-===========
-
-.. automodule:: mlrun.utils
```

Comment:
Are you sure we don't want to document this? We have places where we reference such functions, including the function marketplace itself.

Reply:
u may be mixing it w `mlutils` , this one is all for internal use